src/tests/ftest/control/daos_system_query.yaml

```diff
@@ -1,8 +1,9 @@
 hosts:
   test_servers: 1
+  test_clients: 1
 timeout: 80
 server_config:
-  name: daos_server
+  name: custom_daos_server  # Use a non-default name
```

Comment:
i am not sure this test is a good candidate. i do not see it exercising the agent and client side stack to prove that this works.

Reply:
From a weekly run
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/weekly-testing/379/artifact/Functional%20on%20EL%208/control/daos_system_query.py/job.log

The test runs
- `dmg system query`
- `dmg storage format`
- `daos system query`

Do we need some real IO/app here for the agent+client? Or is `daos system query` enough?

Reply:
ah ok, daos_system_query should be good. i just did not see any agent_config in this yaml file ..

Reply:
The infrastructure will setup an agent config automatically :)

Reply:
this PR has been running for a long time. is it running all VM tests adn do they take that long?

Reply:
It's running `Test-tag: pr,vm DaosSystemQuery` which shouldn't take very long. It looks to me like the VM stage hasn't started? Or maybe Jenkins isn't reporting correctly

Reply:
I agree. Looks like PRs pushed after this one have gone through VM already. Let me try restarting

Reply:
Failed due to some CI issue. triggering again